src/stores/quizStore.ts

```diff
@@ -156,7 +156,15 @@ export const getQuizStore = (set: Function, get: Function) => ({
       return response;
     } catch (err) {
       const response = await getReq('game/data', { gameId });
-      const data = formatGameData(response);
+      const data = formatGameData(
+        response.map((x) => {
+          if (x.SelectedOptionIds) {
+            x.SelectedOptionIds = x.SelectedOptionIds.split(',').map((x) => parseInt(x));
```

Comment:
Can we do this in the backend?

Reply:
showing correct answer and user response side by side